ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py

```diff
@@ -321,12 +325,33 @@ def yield_pipeline(
         """
 
         try:
-            dag: SerializedDAG = self._build_dag(pipeline_details.data)
+            dag_details = self.connection.execute(
+                "SELECT dag.dag_id , dag.fileloc ,serialized_dag.data , dag.max_active_runs , dag.description FROM "
```

Comment:
could we move the queries into a `queries.py` file as with other connectors?

Reply:
yes